user/models.py

```diff
@@ -0,0 +1,83 @@
+from django.db              import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+
+
+class User(models.Model):
+    name       = models.CharField(max_length=45)
+    email      = models.EmailField(max_length=800)
+    phone      = models.CharField(max_length=800)
```

Comment:
phone 을 딱 봤을때 항상 휴대폰 번호라고 확신할 수 없기때문에 이런 부분은 항상 디테일하게 풀어서 지어주세요~
```suggestion
    phone_number      = models.CharField(max_length=800)
```